notifications/fixtures/companiesgroups.json

```diff
@@ -3,29 +3,37 @@
     "model": "notifications.companiesgroup",
     "pk": 1,
     "fields": {
-      "title": "Cars/Vans",
-      "code": "cars-vans"
+      "title": "Cars",
```

Comment:
Better set id for cars and vans  5-6 than changing the ids for the others

Reply:
Done.